### PR TITLE
Adding eThorp as Commercial Support

### PR DESCRIPTION
**Pull request by WalterSchnuble**
I would like to add eThorp Technology as a commercial support resource for the Apache Guacamole project.

eThorp Technology offering Apache Guacamole support in Brazilian Portuguese, providing installation, configuration, optimization, and continuous maintenance to ensure a stable and secure remote desktop experience for businesses seeking reliable DaaS solutions

All the best !

Walter Schnuble

### CODE REVIEW

File: _companies/ethorp.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+title: eThorp technology.
+logo: /images/companies/ethorp.png
+location: https://ethorp.com
+---
+
+eThorp Technology offering Apache Guacamole support in Brazilian Portuguese, providing installation, configuration, optimization, and continuous maintenance to ensure a stable and secure remote desktop experience for businesses seeking reliable DaaS solutions.
```
Comment (mike-jumper) [inline _companies/ethorp.md:7]: I think you probably mean "offers" here (instead of "offering").

**Review by mike-jumper — CHANGES_REQUESTED** [review]
Please also combine these commits into a single, logical commit that describes the high-level nature of the changes being made:

List of three commits, showing unfortunately vague commit messages

The title of this PR is a decent example of such a high-level message.

File: _companies/ethorp.md (hunk 1 above)
Comment (WalterSchnuble) [reply _companies/ethorp.md:7]: Correct. My apologies.
Should we update and make a new pull request?

Comment (mike-jumper) [reply _companies/ethorp.md:7]: Sure, or you can force push to the branch on your fork and it'll get picked up by this existing pull request.

**Review by WalterSchnuble — COMMENTED** [review]
Done

**Comment by necouchman** [comment]
Closed by #153.

**Pull request closed without merging**